ctapipe/reco/hillas_reconstructor.py

```diff
@@ -188,7 +188,7 @@ def __call__(self, event):
         hillas_dict = {
             tel_id: dl1.parameters.hillas
             for tel_id, dl1 in event.dl1.tel.items()
-            if np.isfinite(dl1.tel[tel_id].parameters.hillas.intensity)
+            if np.isfinite(event.dl1.tel[tel_id].parameters.hillas.intensity)
```

Comment:
`dl1.parameters.hillas.intensity`

Reply:
I tried but it doesn't work

Reply:
oh without tels

Reply:
it's the loop variable, you use it also above to fill the dict

Reply:
fixed